Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -304,6 +304,7 @@
   - GreyJumpskirt
   - AncientJumpsuit
   - RainbowJumpsuit
+  - TNGJumpsuit
```

Comment:
you changed the name of the greytider jumpsuit without changing it here

Reply:
WAHH NOoo